source/material_model/rheology/diffusion_creep.cc

```diff
@@ -176,6 +176,14 @@ namespace aspect
                            "Units: \\si{\\meter\\cubed\\per\\mole}.");
         prm.declare_entry ("Grain size", "1e-3", Patterns::Double (0.),
                            "Units: \\si{\\meter}.");
+        prm.declare_entry ("Define phase properties by differences instead of exact values", "true",
```

Comment:
```suggestion
        prm.declare_entry ("Define phase properties by differences instead of exact values", "false",
```
I think the default should be false, so that it is backwards compatible (i.e. input files still have the same meaning).